private/templates/default/config.py

```diff
@@ -693,6 +693,8 @@
 #settings.req.requester_optional = True
 # Uncomment if the User Account logging the Request is NOT normally the Requester
 #settings.req.requester_is_author = False
+# Uncomment to show req_req purpose field in the list view of req_req_skill
+#settings.req_skill_show_purpose = True
```

Comment:
This should be inside the req section, no...?  so settings.req.req_skill_show_purpose ?